src/routes/RemediationDetailsV2.js

```diff
@@ -55,6 +54,20 @@ const RemediationDetailsV2 = () => {
       params: { remId: id },
     }
   );
+
+  const { fetch: getRemediationPlaybookSystems } = useRemediationsQuery(
+    getRemediationPlaybookSystemsList(axios),
+    {
+      skip: true,
+    }
+  );
+  const { fetch: getRemediationPlaybookLogs } = useRemediationsQuery(
+    getPlaybookLogs(axios),
+    {
+      skip: true,
+    }
+  );
+
```

Comment:
1) What is the reason for having these up here and not within the `ExecutionHistoryContent`?

2) The `useEnrichedRuns` and `useLogs` basically replicate what `useRemediationsQuery` already provides.